### PR TITLE
Oz/s1186 plugin fixes

### PR DESCRIPTION
**Pull request by ozhey**
added custom actions

### CODE REVIEW

File: custom_actions/actions/RemoveUser.action.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: "RemoveUser"
+description: "Deletes a user"
+enabled: true
+parameters:
+  "User Email":
+    type: "string"
+    description: " The email of the user, which uniquely identifies the user across all Atlassian products."
```
Comment (shirleygordon-blink) [inline custom_actions/actions/RemoveUser.action.yaml:7]: you have an unnecessary whitespace here :)

**Review by shirleygordon-blink — CHANGES_REQUESTED** [review]
the readme file should be updated as well, some param descriptions should be fixed, for instance AddComment's expand param: "Use [expand](#expansion) to include..."
also the links in the readme aren't formatted properly

File: custom_actions/actions/RemoveUser.action.yaml (hunk 1 above)
Comment (ozhey) [reply custom_actions/actions/RemoveUser.action.yaml:7]: hawkeye

Comment (YakirOren) [reply custom_actions/actions/RemoveUser.action.yaml:7]: `yq e -i custom_actions/actions/RemoveUser.action.yaml`

**Comment by ozhey** [comment]
> the readme file should be updated as well, some param descriptions should be fixed, for instance AddComment's expand param: "Use [expand](#expansion) to include..." also the links in the readme aren't formatted properly

done

**Review by shirleygordon-blink — APPROVED** [review]
great job :)

**Pull request merged**